src/Github/Api/V3/CreatePullRequest.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\AutomaticReleases\Github\Api\V3;
+
+use Assert\Assert;
+use Doctrine\AutomaticReleases\Git\Value\BranchName;
+use Doctrine\AutomaticReleases\Github\Value\RepositoryName;
+use Psr\Http\Client\ClientInterface;
+use Psr\Http\Message\RequestFactoryInterface;
+
+final class CreatePullRequest
+{
+    private const ENDPOINT = 'https://api.github.com/';
```

Comment:
This isn't enpoint but API root. Maybe:
```suggestion
    private const ENDPOINT = 'https://api.github.com/repos/%s/%s/pulls';
```